app/src/main/java/de/westnordost/streetcomplete/screens/settings/questselection/QuestPresetsAdapter.kt

```diff
@@ -115,6 +120,10 @@ class QuestPresetsAdapter(
                 renameItem.setOnMenuItemClickListener { onClickRenamePreset(preset); true }
             }
 
+            val duplicateItem = popup.menu.add(R.string.quest_presets_duplicate)
+            duplicateItem.setIcon(R.drawable.ic_add_24dp)
```

Comment:
I'd rather use the `copy` icon: https://pictogrammers.com/library/mdi/icon/content-copy/

Reply:
Fixed ;)